.github/workflows/enforceTestCoverage.yml

```
@@ -35,20 +35,17 @@
-          else
-            echo "solidity_changed=false" >> "$GITHUB_ENV"
-          fi
+          echo "solidity_changed=true" >> "$GITHUB_ENV"
 
       - name: Set up Node.js
         uses: actions/setup-node@v4.1.0
         with:
           node-version: '20'
         if: env.solidity_changed == 'true'
 
+      - name: Set up Bun
```

actionlint (1.7.4)

step must run script with "run" section or run action with "uses" section

(syntax-check)